desktop/components/auth_modal/view.coffee

```diff
@@ -39,12 +39,13 @@ module.exports = class AuthModalView extends ModalView
     super
 
   preInitialize: (options = {}) ->
-    { @copy } = options
+    { @copy, @context } = options
     @user = new LoggedOutUser
     mode = mode: options.mode if options.mode
     @state = new State mode
 
     @templateData = _.extend {
+      context: @context
```

Comment:
This'll pass `context` value down to the template, so we can specify a context for the facebook auth button.